docs/docs-content/clusters/cluster-management/backup-restore/create-cluster-backup.md

```diff
@@ -161,6 +170,14 @@ months. Additionally, you can initiate a backup on demand for an existing cluste
       deletionPolicy: "Retain"
    ```
 
+   Additionally, you must add the following snippet under the `manifests.volume-snapshot-class` field if you are using <VersionedLink text="Portworx" url="/integrations/packs/?pack=csi-portworx-generic" /> as your CSI layer on a cluster deployed to a MAAS environment. These labels ensure that the <VersionedLink text="Volume Snapshot Controller" url="/integrations/packs/?pack=volume-snapshot-controller" /> pack installs correctly.
```

Comment:
@addetz this awesome content. Any chance you can also add this to the Additional Details tab of the two portworx packs plz?

Reply:
Excellent shout! Done this now.